src/screens/Government/Government.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  Wrapper,
+  Content,
+  TextContent,
+  Header,
+  EmailButton,
+} from './Government.style';
+
+const Government = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <Wrapper>
+      <Header>
+        <Content>
+          <Typography variant="h3" component="h1">
+            Government Partners
+          </Typography>
+        </Content>
+      </Header>
+      <Content>
+        <TextContent>
+          <Typography variant="h5" component="h5">
+            Are you with a state or local government?
+          </Typography>
+          <Typography variant="body1" component="p">
+            Our goal is to provide you with data, modeling, and analysis to help
+            your government manage its response to COVID. Specifically you can:
+          </Typography>
+          <ol>
+            <li>
+              <Typography variant="body1" component="p">
+                <strong>
+                  Work with us to improve the data inputs for your state- or
+                  county-specific COVID model
+                </strong>{' '}
+                <br />
+                Our model is only as good as its data. If you have different or
+                higher quality data, we can work together to ingest that data
+                into the model and create better, more accurate COVID
+                projections to help your government plan its response. Connect
+                with us at{' '}
+                <a href="mailto:gov@covidactnow.org">gov@covidactnow.org</a>.
+              </Typography>
+            </li>
+            <li>
+              <Typography variant="body1" component="p">
+                <strong>Use our API</strong> <br />
+                You can find our API{' '}
+                <a href="https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md">
+                  here
+                </a>
```

Comment:
For some reason the link isn't showing up properly on the preview:

![image](https://user-images.githubusercontent.com/2575327/79680380-36beb200-81cc-11ea-8435-4e675d3a337f.png)

Reply:
Updated! Link was old.